lib/feature/one_hot_encoder.cc

```diff
@@ -14,36 +14,70 @@
  * limitations under the License.
  */
 
-#include "fstream"
-#include "sys/stat.h"
+#include "clink/feature/one_hot_encoder.h"
+
 #include <dirent.h>
-#include <string>
+#include <fstream>
+#include <sys/stat.h>
 
-#include "clink/feature/one_hot_encoder.h"
 #include "nlohmann/json.hpp"
-#include "tfrt/support/error_util.h"
 
-using namespace clink;
-using namespace std;
-using namespace nlohmann;
+namespace clink {
+
+namespace {
+
+// Given a directory path, gets the only file in the directory.
+//
+// Flink ML saves model data in a file whose name is unknown to C++. This
+// function helps to locate that file.
+//
+// This function returns empty string if the directory does not exist, or there
+// is zero or more than one file in the directory.
+std::string getOnlyFileInDirectory(std::string path) {
```

Comment:
This method will likely be useful to other algorithms as well. Could we move it e.g. `clink_utils.cc`?